src/Akka.Persistence.Sql/snapshot.conf

```diff
@@ -30,10 +30,8 @@
 
       dao = "Akka.Persistence.Sql.Snapshot.ByteArraySnapshotDao, Akka.Persistence.Sql"
 
-      compatibility-mode = false
-
       # if true, tables will attempt to be created.
-      auto-initialize = false
+      auto-initialize = true
```

Comment:
Persistence plugins should work as is out of the box, database tables should be initialized automatically.
This is open for discussion, should this be reverted?

Reply:
Makes sense to me, I start greenfield from an empty db